python_modules/dagster-airflow/dagster_airflow/dagster_plugin.py

```diff
@@ -29,6 +31,23 @@
 # We don't use six here to avoid taking the dependency
 STRING_TYPES = ("".__class__, u"".__class__)
 
+# Again, we don't use six
```

Comment:
I guess this is safer than counting on the transitive dependency implied by dagster. also that's hilarious that this is the implementation

Reply:
eh this is actually me being maybe a little misguided and still trying to support the "drop this one file in your plugins directory and don't take any dependencies" approach

Reply:
like, still formally supporting the scaffolded version

Reply:
oic. it seems like overkill to me but I'll defer to you and @natekupp on this one